docs/reference/mapping/types/ip.asciidoc

```diff
@@ -45,6 +45,11 @@ The following parameters are accepted by `ip` fields:
 
 [horizontal]
 
+<<dimension,`dimension`>>::
+
+    This parameter marks the field as a time series dimension field. Accepts
+    `true` or `false` (default).
```

Comment:
Removing the `<<...>>` syntax should fix the docs build.
```suggestion
`dimension`::

    Marks the field as a time series dimension. Accepts `true` or `false`
    (default).
```

Reply:
Thanks a lot James!